core/src/main/java/org/arquillian/smart/testing/spi/TestResultParser.java

```diff
@@ -0,0 +1,25 @@
+package org.arquillian.smart.testing.spi;
+
+import java.io.InputStream;
+import java.util.Set;
+
+/**
+ * Java SPI for parsing test results stored in a stream (probably a file).
```

Comment:
`(probably a file)` - not sure how end-user would understand that part. What do we want to say here?